### PR TITLE
Expanded client policy support in asbench

### PR DESCRIPTION
**Pull request by dotyjim-work**
https://aerospike.atlassian.net/browse/PROD-1964

### CODE REVIEW

**Comment by dotyjim-work** [comment]
Suggest we drop db08d891952b393710e8d4d1f26a6a97d33c7d74 before merging, but it was rather useful to leverage the github actions build step.

I did some manual testing, but didn't invest in getting the test suite running locally.

File: src/main/benchmark_init.c [hunk 1]
```diff
@@ -98,12 +114,17 @@ static struct option long_options[] = {
...
+	{"connect-timeout",       required_argument, 0, CONNECT_TIMEOUT}, // flag matches Java Benchmark, but controls conn_timeout_ms
 	{"read-socket-timeout",   required_argument, 0, '2'},
 	{"write-socket-timeout",  required_argument, 0, '3'},
 	{"timeout",               required_argument, 0, 'T'},
 	{"read-timeout",          required_argument, 0, 'X'},
 	{"write-timeout",         required_argument, 0, 'V'},
 	{"max-retries",           required_argument, 0, 'r'},
+	{"max-error-rate",        required_argument, 0, MAX_ERROR_RATE},
+	{"tender-interval",       required_argument, 0, TENDER_INTERVAL},
+	{"error_rate_window",     required_argument, 0, ERROR_RATE_WINDOW},
```
Comment (ClaytonKnittel) [inline src/main/benchmark_init.c:126]: This should be dash-delimited

File: src/main/benchmark_init.c [hunk 2]
```diff
@@ -1157,6 +1337,22 @@ set_args(int argc, char * const* argv, args_t* args)
 				args->max_retries = atoi(optarg);
 				break;
 
+			case 'E':
```
Comment (ClaytonKnittel) [inline src/main/benchmark_init.c:1340]:
```suggestion
			case MAX_ERROR_RATE:
```

File: src/main/benchmark_init.c (hunk 1 above)
Comment (dotyjim-work) [reply src/main/benchmark_init.c]: [6dec2be](https://github.com/aerospike/aerospike-benchmark/pull/67/commits/6dec2be036d5908953480991fc04f5f8ed6b7bf0)

**Comment by ClaytonKnittel** [comment]
Replaced by https://github.com/aerospike/aerospike-benchmark/pull/68

**Pull request closed without merging**